R/class2tree.R

```diff
@@ -31,6 +31,17 @@
 #' @details See [vegan::taxa2dist()]. Thanks to Jari Oksanen for
 #' making the taxa2dist function and pointing it out, and Clarke & Warwick
 #' (1998, 2001), which taxa2dist was based on.
+#' The taxonomy tree created is not only based on the clustering of the taxonomy
+#' ranks (e.g. strain, species, genus, ...), but it also utilizes the actual 
+#' taxon clades (e.g. mammals, plants or reptiles, etc.). The process of this 
+#' function is as following: First, all possible taxonomy ranks and their 
+#' corresponding IDs for each given taxon will be collected from the input.
+#' Then, the rank vectors of all taxa will be aligned, so that they together 
+#' will become a matrix with columns are a list of ordered taxonomy ranks of all
```

Comment:
So each column is a list. Is that true? Like a nested list within each cell?

Reply:
uhm, no, what I want to say is: the column header is a ordered list of all available taxonomy ranks. Each cell has only one value.
What should I write here to be more precise or be easier to understand? :-P

Reply:
i assume "ordered" here means the same as sorted? maybe "where columns are ordered taxonomy ranks of all taxa" would work?

Reply:
yes, "ordered" means "sorted" :)